core/src/test/java/org/infinispan/persistence/ClusteredConditionalCommandTest.java

```diff
@@ -75,6 +78,18 @@ private ConfigurationBuilder createConfiguration(String storeName, boolean share
       return builder;
    }
 
+   @AfterMethod
+   public void afterMethod() {
+      if (passivation) {
+         for (EmbeddedCacheManager cacheManager : cacheManagers) {
+            ActivationManager activationManager = TestingUtil.extractComponent(cacheManager.getCache(PRIVATE_STORE_CACHE_NAME),
+                  ActivationManager.class);
+            // Make sure no passivations are pending, which could leak between tests
```

Comment:
Nitpick: You're checking for activations, not passivations

Reply:
Yeah meant activations.